vimoutliner/plugin/votl_smart_paste.vim

```diff
@@ -4,10 +4,10 @@
 " middle of a branch.
 " Added 2011-03-01(JB): This script will now also copy an outline correctly by
 " using \\y, copy to the register with \\r, cut an outline by using \\d, and
-" paste from the register using \\b. 
+" paste from the register using \\b.
 " http://www.lists.vimoutliner.org/pipermail/vimoutliner/2008-October/002366.html
 
-map <buffer>p :call VOput()<cr>
```

Comment:
1. How much it contributes to the fixing of the bug “Fix VOput() bug reported by grahameblackwood”?
2. It seems to me like you are second-guessing the current design of vim itself, not VO. I guess if you do that, you should get three times the content of "* register pasted. And so what?

Please, remove this chunk.